MetaMask/metamask-extension
# fix: #25189 - Display network badge for send flow asset

## **Description**

Adds the network badge for the chosen asset in the new Send flow

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/25470?quickstart=1)

## **Related issues**

Fixes: https://github.com/MetaMask/metamask-extension/issues/25189

## **Manual testing steps**

1. Go to Ethereum Mainnet
2. Go to the Send flow
3. See the chosen asset with Mainnet logo
4. Change to Base network
5. Go to Send flow
6. See the chosen asset with the Base logo

## **Screenshots/Recordings**



### **Before**

<img width="409" alt="338344947-c9a6eff0-74f1-4b8a-bca4-48a8325a14d8" src="https://github.com/MetaMask/metamask-extension/assets/46655/856305e3-5858-4d26-9280-7b539250de6e">



### **After**

<img width="421" alt="SCR-20240621-jquj" src="https://github.com/MetaMask/metamask-extension/assets/46655/446dc321-5f62-4287-8ab7-5d2ac0eecbd6">




## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
